test/acceptance_tests/test_cropped_decoding.py

```diff
@@ -14,12 +14,111 @@
 
 from braindecode.datautil.iterators import CropsFromTrialsIterator
 from braindecode.models.util import to_dense_prediction_model
-from braindecode.monitors import compute_preds_per_trial_from_crops
 from braindecode.models import ShallowFBCSPNet
 from braindecode.util import set_random_seeds
 from braindecode.util import var_to_np, np_to_var
 
 
+def compute_preds_per_trial_from_crops(all_preds, input_time_length, X):
```

Comment:
Actually whole test should be refactored to new dataset etc. classes and then use existing functions from scoring, if not possible, refactor scoring to make the functionality accessible (for the functions now copied inside from old monitor.py). Wanna have a try @sliwy ?

Reply:
I've opened an issue #97 for this.

Reply:
siüer!